src/main/java/ch/njol/skript/expressions/ExprRound.java

```diff
@@ -39,10 +39,9 @@
  * @author Peter Güttinger
  */
 @Name("Rounding")
-@Description("Rounds numbers normally, up (ceiling) or down (floor) respectively")
+@Description("Rounds numbers normally, up (ceiling) or down (floor) respectively.")
 @Examples({"set {var} to rounded health of player",
-		"set line 1 of the block to round(1.5 * player's level)",
-		"set {_x} to floor({_y}) - ceil({_x})",
+		"set line 1 of the block to rounded (1.5 * player's level)",

```

Comment:
```suggestion
		"set line 1 of the block to "%rounded (1.5 * player's level)%"",
```

Reply:
Wouldn't some existing number -> string converter make it work anyway?

Reply:
It doesn't exist sadly

Reply:
Hmm... I assume it wouldn't be exactly good to have everything auto-converted to Strings when needed, but wouldn't it be good to have numbers do that? :eyes: It's kinda a basic conversion IMO

Reply:
_Sure, if you wanna break Skript, go ahead. /s_

Joke aside, it's know that String converters fuck up things more than what it is expected. I'd say Skript is fine without this feature (mainly since I never liked it, doesn't make sense at all).